nixos/tests/virtualbox.nix

```diff
@@ -314,10 +323,10 @@ let
     ${pkgs.dhcp}/bin/dhclient \
       -lf /run/dhcp.leases \
       -pf /run/dhclient.pid \
-      -v eth0 eth1
+      -v enp0s3 enp0s8
```

Comment:
Note that this should be inside the guest without `systemd`, so there shouldn't be predictable interface names. Did you actually test this?

Reply:
Stage 1 runs udev, which renames the interfaces, so it seems something has changed along the way.

However, I realized this can be changed to just `networking.usePredictableInterfaceNames = false` in `testVMConfig` above.